### DISCUSSION TITLE
serialEvent() does not seem to work or is not implemented

### DISCUSSION THREAD

**Discussion by cyborgdennett** [bxparks/UnixHostDuino#67] [General]
I wanted to use: [serialEvent Arduino ](https://www.arduino.cc/reference/en/language/functions/communication/serial/serialevent/)

Taking the '''StdioSerialEcho.ino''' example as base.

Changed the following lines:

```
void serialEvent(){
  loopImplicitly();
}

void loop() {
  // loopExplicitly();
}
```

I would expect this to read the keyboard input, but it doesn't.

**Comment by bxparks** [comment]
Unfortunately EpoxyDuino does not support `serialEvent()`. I've never used this functionality, and don't even know why it would be useful. The website that you link contains a number of boards for which `serialEvent()` is not implemented, so it's not even universally available.

Can you not call `loopImplicitly()` (or the equivalent function that you want to implement) explicitly in your `loop()` method?
